__tests__/app.test.js

```diff
@@ -71,8 +71,70 @@ describe("GET /api/reviews", () => {
         });
       });
   });
-});
+  test("only returns reviews of a given category when queried by category", () => {
+    return request(app)
+      .get("/api/reviews?category=dexterity")
+      .then(({ body }) => {
+        const { reviews } = body;
+        reviews.forEach((review) => {
+          expect(review.category).toBe("dexterity");
+        });
+      });
+  });
+  it("should return an empty array if the given category query has no reviews", () => {
+    return request(app)
+      .get("/api/reviews?category=test")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toEqual([]);
+      });
+  });
+  test("sorts by a given column name if the sort_by query is provided", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=comment_count")
```

Comment:
Here you are only testing that one of your included set_by queries is passing, if you wanted to check all of them there re ways to test this nicely, one is to create an array of promises with all of the appropriate queries and make sure non of them 400